src/runtime_src/core/pcie/driver/linux/xocl/subdev/icap.c

```diff
@@ -1111,6 +1111,10 @@ static int icap_download_hw(struct icap *icap, const struct axlf *axlf)
 		primaryFirmwareOffset = primaryHeader->m_sectionOffset;
 		primaryFirmwareLength = primaryHeader->m_sectionSize;
 	}
+        else{
+                ICAP_ERR(icap,"invalied xclbin failed to read bitstream");
```

Comment:
fix the typo. *Invalid*
You can reword something like "Invalid xclbin. Bitstream is not present in xclbin"